src/hooks/useVerifiableCredential.ts

```diff
@@ -31,7 +26,7 @@ export type VCSignError = {
  */
 export function useVerifiableCredential() {
   const verifyClient = usePublicClient();
-  const chainId = useChainId();
+  const chainId = mainnet.id; // TODO: use useChainId() when we start allowing multiple chains
```

Comment:
you dont need that when u change wagmi-config.ts

Reply:
fixed